.github/workflows/deploy.yml

```diff
@@ -21,6 +21,11 @@ jobs:
       - name: Import the nested docs
         run: make import
 
+      - name: Generate newer usage graphs
+        run: |
+          pip3 install click requests pygal
+          python3 files/generate_usage.py
```

Comment:
Nice! I don't need to update manually..;)

Reply:
It will be a bit messy when running locally. It seems to run with each `make run-dev`, maybe we'll need to adjust it a bit.